includes/wccom-site/rest-api/v1/class-wc-rest-wccom-site-installer-v1-controller.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * WCCOM Site Installer REST API Controller
+ *
+ * Handles requests to /installer.
+ *
+ * @package WooCommerce/WCCOM_Site/REST_API
+ * @since   3.7.0
+ */
+
+defined( 'ABSPATH' ) || exit;
```

Comment:
This style should be used with all new files.

Reply:
Yep. All new files have that.